zeppelin-web/src/app/editor-configuration/editor.configuration.controller.js

```diff
@@ -0,0 +1,231 @@
+    $scope.updateEditorSettings();
+  };
+
+  $scope.changeShowFocusLine = function(isShowActiveLine) {
+    $scope.isChangedConfiguration(true);
+    $scope.modifyTargetEditorSetting.setting.activeLine = isShowActiveLine;
+    $scope.updateEditorSettings();
+  };
+
+  $scope.changeLiveAutoCompletion = function(isLiveAutoCompletion) {
+    $scope.isChangedConfiguration(true);
+    $scope.modifyTargetEditorSetting.setting.liveAutoCompletion = isLiveAutoCompletion;
+    $scope.updateEditorSettings();
+  };
+
+  $scope.changeShowPrintMargin = function(isShowPrintMargin) {
+    $scope.isChangedConfiguration(true);
+    $scope.modifyTargetEditorSetting.setting.showPrintMargin = isShowPrintMargin;
+    $scope.updateEditorSettings();
+  };
+
+  $scope.changeShowPrintMarginColumn = function(marginColumn) {
+    $scope.isChangedConfiguration(true);
+    $scope.modifyTargetEditorSetting.setting.showPrintMarginColumn = marginColumn;
+    $scope.updateEditorSettings();
+  };
+
+  $scope.saveConfiguration = function() {
+    $scope.isChangedConfiguration(false);
+    $scope.restoreEditorConfig.userConfig = angular.copy($scope.modifyTargetEditorSetting);
+    websocketMsgSrv.saveUserCodeEditorSetting(editorConfigSrv.getEditorConfigLists());
+  };
+
+  $scope.restoreConfiguration = function() {
+    if ($scope.selectedEditorConfigIndex !== undefined && $scope.restoreEditorConfig !== undefined) {
+      editorConfigSrv.setEditorConfig($scope.selectedEditorConfigIndex, $scope.restoreEditorConfig);
+      $scope.updateEditorSettings();
+    }
+  };
+
+  $scope.changeEditorTargetFromName = function(editorModeName) {
+    var config = editorConfigSrv.findGetEditorConfigFromName(editorModeName);
+    if (config !== undefined) {
+      $scope.restoreEditorConfig = angular.copy(config.data);
+      $scope.selectedEditorConfigIndex = config.index;
+      $scope.modifyTargetEditorSetting = config.data.getUserConfig();
+    }
+  };
+
+  $scope.changeEditorTargetFromIndex = function(index) {
+    var config = editorConfigSrv.getEditorConfig(index);
+    if (config !== undefined) {
+      $scope.restoreEditorConfig = angular.copy(config);
+      $scope.selectedEditorConfigIndex = index;
+      $scope.modifyTargetEditorSetting = config.getUserConfig();
+    }
+  };
+
+  $scope.showRemoveConfigurationDialog = function() {
+    BootstrapDialog.confirm({
+      closable: true,
+      type: BootstrapDialog.TYPE_DANGER,
+      title: 'DELETE',
+      message: 'Do you want to delete this editor configuration?',
+      callback: function(result) {
+        if (result) {
+          $scope.isChangedConfiguration(true);
+          $scope.removeConfiguration($scope.selectedEditorConfigIndex);
+        }
+      }
+    });
+  };
+
+  $scope.showCreateConfigurationDialog = function() {
+    BootstrapDialog.show({
+      title: 'Please Input to New &lt;User Editor Setting&gt; Name',
+      message: jQuery('<form> ' +
```

Comment:
@corneadoug @bzz 
I refer to the following guidelines.
https://nakupanda.github.io/bootstrap3-dialog/
Rich message part.

In the example, I see that you are using Jquery.